gallery/plot_transforms.py

```diff
@@ -89,3 +89,65 @@ def _plot(img, title, **kwargs):
 # performs random affine transform on an image.
 affined_img = T.RandomAffine(degrees=(30, 70), translate=(0.1, 0.3), scale=(0.5, 0.75))(orig_img)
 plot(affined_img, "Affine transformed image")
+
+####################################
+# RandomCrop
+# ------------
+# The :class:`~torchvision.transforms.RandomCrop` transform
+# (see also :func:`~torchvision.transforms.functional.crop`)
+# performs random crop transform on an image.
+randomcrop_img = T.RandomCrop(size=(128, 128))(orig_img)
```

Comment:
nitpick: here and in other places:

```suggestion
random_crop_img = T.RandomCrop(size=(128, 128))(orig_img)
```

Reply:
Thanks @NicolasHug done for all the variables as per your suggestion